log/logrus_logger.go

```diff
@@ -0,0 +1,34 @@
+package log
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/sirupsen/logrus"
```

Comment:
We don't want the base Go kit `log` package to have a dependency on logrus. The logrus adapter should be in its own package in a subdirectory. I suggest it go in `github.com/go-kit/kit/log/logrus`.